src/components/Blocking/SpecificIssue.js

```diff
@@ -90,12 +99,15 @@ const styles = StyleSheet.create({
     paddingRight: 13,
     paddingLeft: 19,
     fontFamily: fonts.inter[400],
-    color: COLORS.blackgrey
+    color: COLORS.white2,
+    borderWidth: 1,
+    borderColor: COLORS.gray200
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the text color and border color are acceptable as long as they maintain good contrast with the background for readability and accessibility. 



<!-- This is an auto-generated comment by OSS CodeRabbit -->